### PR TITLE
Add getQuota method using imap_get_quotaroot to Connection class

### PR DESCRIPTION
**Pull request by arkadiusjonczek**
As discussed in https://github.com/ddeboer/imap/pull/421 I changed the requested code lines.
(Sorry I had to create a new PR because I deleted the old repository)

### CODE REVIEW

**Comment by arkadiusjonczek** [comment]
Would be nice if you could add a new release and update the composer package after merge. So I can use the changes in my application. 👍 :)

**Comment by arkadiusjonczek** [comment]
Waiting for your feedback @ddeboer @Slamdunk ;)

File: tests/ConnectionTest.php [hunk 1]
```diff
@@ -63,6 +63,15 @@ public function testPing()
         $connection->ping();
     }
 
+    public function testQuota()
+    {
+        if (false !== \getenv('IMAP_QUOTAROOT_SUPPORTED')) {
```
Comment (Slamdunk) [inline tests/ConnectionTest.php]: 1. Please swap the logic here: if getenv is false, mark test skipped.
1. Could you please try to install the quota plugin in travis Dovecot installation? So we can prove your code works as expected:
    1. https://wiki2.dovecot.org/Quota/Configuration
    1. https://github.com/ddeboer/imap/blob/master/.travis/dovecot_install.sh

**Comment by Slamdunk** [comment]
Note to self: working good against GMail:

```
array(3) {
  ["usage"]=>
  int(4665144)
  ["limit"]=>
  int(19922944)
  ["STORAGE"]=>
  array(2) {
    ["usage"]=>
    int(4665144)
    ["limit"]=>
    int(19922944)
  }
}
```

File: tests/ConnectionTest.php (hunk 1 above)
Comment (arkadiusjonczek) [reply tests/ConnectionTest.php]: I updated the test case logic and enabled quota plugin in travis dovecot installation. Seems to work fine. Please review again. ;)

**Comment by Slamdunk** [comment]
Awesome work :+1: thank you

**Comment by Slamdunk** [comment]
Released in [1.9.0](https://github.com/ddeboer/imap/releases/tag/1.9.0)

**Comment by arkadiusjonczek** [comment]
Thank you my friend @Slamdunk 👍

**Pull request merged**